central/graphql/resolvers/common_vulnerabilities.go

```diff
@@ -33,7 +33,6 @@ var commonVulnerabilitySubResolvers = []string{
 	"suppressed: Boolean!",
 	"unusedVarSink(query: String): Int",
 	"vectors: EmbeddedVulnerabilityVectors",
-	"vulnerabilityState: String!",
```

Comment:
Why is this being pulled out and added to just node and image vulns? Is it not appropriate for cluster?

Reply:
I think it is only appropriate for image vulns and is not needed for cluster and node vulns

Reply:
Yes, `vulnerabilityState` is only supported for image vulns.